examples/Consumer/Program.cs

```diff
@@ -174,16 +192,57 @@ public static void Run_Consume(string brokerList, List<string> topics)
 
                     if (msg.Offset % 5 == 0)
                     {
-                        Console.WriteLine($"Committing offset");
                         var committedOffsets = consumer.CommitAsync(msg).Result;
                         Console.WriteLine($"Committed offset: {committedOffsets}");
                     }
                 }
             }
         }
 
+        /// <summary>
+        ///     In this example
+        ///         - consumer group functionality (i.e. .Subscribe + offset commits) is not used.
+        ///         - the consumer is manually assigned to a partition and always starts consumption
+        ///           from a specific offset (0).
```

Comment:
offset 0 is problematic since it can be invalid due to retention, in which case it will start consuming the Newest message instead.

Better to explicitly assign Beginning